### PR TITLE
CIVDT-82: Customs Duty page

### PR DESCRIPTION
**Pull request by tvaidotas**

### CODE REVIEW

**Review by macgrewal — CHANGES_REQUESTED** [review]
You are missing tests for your form and as such have missed the scenario where a user enters a numeric value with more than 2 decimal places such as 11.111. This scenario results in an error message `error.invalidNumeric`

Scenario 4 in the story has not been met (original amount > amended amount)

File: test/forms/CustomsDutyFormProviderSpec.scala [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  "Binding a form with invalid data" when {
+
+    "no values provided" should {
+      "result in a form with errors" in {
+        formBinder().errors mustBe Seq(
+          FormError(originalKey, originalNonEmptyMessageKey),
+          FormError(amendedKey, amendedNonEmptyMessageKey)
+        )
+      }
+
```
Comment (macgrewal) [inline test/forms/CustomsDutyFormProviderSpec.scala:53]: You are missing a `}` (then remove the extra space from the end of the file). In its current state, the output of the unit tests reads incorrectly.

File: test/controllers/CustomsDutyControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      charset(result) mustBe Some("utf-8")
+    }
+
+  }
+
+  "POST /" when {
+
+    "should redirect to Import VAT page" in new Test {
+      controller.redirect(Some(UnderpaymentType(true, true, false))) mustBe Redirect(controllers.routes.CustomsDutyController.onLoad()) // Import VAT
+    }
```
Comment (macgrewal) [inline test/controllers/CustomsDutyControllerSpec.scala:91]: Please rewrite this and other navigation tests like this:
```suggestion
    "Import VAT was selected as an underpayment" should {
      "redirect to Import VAT page" in new Test {
        controller.redirect(Some(UnderpaymentType(true, true, false))) mustBe 
  Redirect(controllers.routes.CustomsDutyController.onLoad()) // Import VAT
        }
      }
```
Current output:
```
[info] POST /
[info] - when should redirect to Import VAT page
[info] - when should redirect to Excise Duty page
[info] - when should redirect to Summary page
[info]   when payload contains valid data
[info]   - should return a SEE OTHER response when correct data is sent
[info]   - should update the UserAnswers in session
[info]   when payload contains invalid data
[info]   - should return BAD REQUEST when original amount is exceeded
[info]   - should return BAD REQUEST
```
Expected output:
```
[info] POST /
[info]   when Import VAT was selected as an underpayment
[info]   - should redirect to Import VAT page
[info] - when should redirect to Excise Duty page
[info] - when should redirect to Summary page
```

**Review by macgrewal — APPROVED** [review]

**Review by macgrewal — CHANGES_REQUESTED** [review]
Sorry meant to say please make the suggested changes, not approved.

File: test/forms/CustomsDutyFormProviderSpec.scala (hunk 1 above)
Comment (tvaidotas) [reply test/forms/CustomsDutyFormProviderSpec.scala:53]: this comment doesn't make since the tests for invalid input are up to line ~97

**Pull request merged**